update.go

```diff
@@ -19,6 +20,15 @@ eiFH+GffIMk09RFqkcMh5rIPu3ykm5V8
 `)
 
 func update(channel string, silent bool) error {
+	now := time.Now()
+	equinoxEOL, _ := time.Parse(time.RFC3339, "2021-09-30T00:00:00Z")
+	if now.After(equinoxEOL) {
+		if !silent {
+			log.Println("Equinox has shut down, update functionality has been disabled")
```

Comment:
Maybe this would be a good place to link back here, and or to docs on how to update it themselves?